lib/fastlane/plugin/wpmreleasetoolkit/helper/git_helper.rb

```diff
@@ -124,7 +124,18 @@ def self.list_local_tags(matching: '*')
       # @param [Bool] delete_on_remote If true, will also delete the tag from the remote. Otherwise, it will only be deleted locally.
       #
       def self.delete_tags(tag_names, delete_on_remote: false)
-        Action.sh('git', 'tag', '-d', *tag_names)
+
+        tag_names = [tag_names] unless tag_names.is_a? Array
+
+        g = Git.open(Dir.pwd)
+        local_tag_names = g.tags.map do |tag|
+          tag.name
+        end
```

Comment:
```suggestion
        local_tag_names = g.tags.map(&:name)
```

Reply:
Right – this is much nicer. Addressed in d3d0d673deebb9a288050efa0da9e92bdb279403